test/tools/libtesteth/TestHelper.cpp

```diff
@@ -67,531 +66,540 @@ void connectClients(Client& c1, Client& c2)
+    case json_spirit::bool_type:
+        return (u256)(uint64_t)_v.get_bool();
+    case json_spirit::real_type:
+        return (u256)(uint64_t)_v.get_real();
+    default:
+        cwarn << "Bad type for scalar: " << _v.type();
+    }
+    return 0;
 }
 
 byte toByte(json_spirit::mValue const& _v)
 {
-	switch (_v.type())
-	{
-	case json_spirit::str_type: return (byte)stoi(_v.get_str());
-	case json_spirit::int_type: return (byte)_v.get_uint64();
-	case json_spirit::bool_type: return (byte)_v.get_bool();
-	case json_spirit::real_type: return (byte)_v.get_real();
-	default: cwarn << "Bad type for scalar: " << _v.type();
-	}
-	return 0;
+    switch (_v.type())
+    {
+    case json_spirit::str_type:
+        return (byte)stoi(_v.get_str());
+    case json_spirit::int_type:
+        return (byte)_v.get_uint64();
+    case json_spirit::bool_type:
+        return (byte)_v.get_bool();
+    case json_spirit::real_type:
+        return (byte)_v.get_real();
+    default:
+        cwarn << "Bad type for scalar: " << _v.type();
+    }
+    return 0;
 }
 
 bytes importByteArray(string const& _str)
 {
-	checkHexHasEvenLength(_str);
-	return fromHex(_str.substr(0, 2) == "0x" ? _str.substr(2) : _str, WhenError::Throw);
+    checkHexHasEvenLength(_str);
+    return fromHex(_str.substr(0, 2) == "0x" ? _str.substr(2) : _str, WhenError::Throw);
 }
 
 bytes importData(json_spirit::mObject const& _o)
 {
-	bytes data;
-	if (_o.at("data").type() == json_spirit::str_type)
-		data = importByteArray(_o.at("data").get_str());
-	else
-		for (auto const& j: _o.at("data").get_array())
-			data.push_back(toByte(j));
-	return data;
+    bytes data;
+    if (_o.at("data").type() == json_spirit::str_type)
+        data = importByteArray(_o.at("data").get_str());
+    else
+        for (auto const& j : _o.at("data").get_array())
+            data.push_back(toByte(j));
+    return data;
 }
 
 void replaceLLLinState(json_spirit::mObject& _o)
 {
-	for (auto& account: _o.count("alloc") ? _o["alloc"].get_obj() : _o.count("accounts") ? _o["accounts"].get_obj() : _o)
-	{
-		auto obj = account.second.get_obj();
-		if (obj.count("code") && obj["code"].type() == json_spirit::str_type)
-		{
-			string code = obj["code"].get_str();
-			obj["code"] = compileLLL(code);
-		}
-		account.second = obj;
-	}
-}
-
-vector<fs::path> getFiles(fs::path const& _dirPath, set<string> const _extentionMask, string const& _particularFile)
-{
-	vector<fs::path> files;
-	for(auto const& ext: _extentionMask)
-	{
-		if (!_particularFile.empty())
-		{
-			fs::path file = _dirPath / (_particularFile + ext);
-			if (fs::exists(file))
-				files.push_back(file);
-		}
-		else
-		{
-			using fsIterator = fs::directory_iterator;
-			for (fsIterator it(_dirPath); it != fsIterator(); ++it)
-			{
-				if (fs::is_regular_file(it->path()) && it->path().extension() == ext)
-					files.push_back(it->path());
-			}
-		}
-	}
-	return files;
+    for (auto& account : _o.count("alloc") ? _o["alloc"].get_obj() :
```

Comment:
This looks bad. We have 3 `:` here. Can you move the "accounts" initialization outside of the loop?